### PR TITLE
Make homepage a faithful representation of mocks

### PR DESCRIPTION
**Pull request by filmaj**
Re: adobe/developer.adobe.com-planning#126

Work in progress! Feel free to creep / review / leave comments.

# TODO

## Mobile View

- [ ] no real idea on how to handle the background images yet
- [x] the hamburger button needs to be positioned properly
- [x] there is a white margin / border around the whole page that needs to go - seems related to the `spectrum-grid--fluid` and `spectrum-grid-row` classes. `fluid` wants 16px padding on each side, whereas `row` sets a negative -8px margin on each side, so there's an 8px gap.
- [x] how to do the white bar indents in the authentication guides?
- [ ] how will we do the product spotlight carousel?
- [x] the recent blog articles contain a 'length of read'. this is not provided by the medium api. however, we are given the full text content of the article. could we estimate the read length based on that?
- [x] would be good to fix the current TODO in the blog article code around making sure we don't cut off unclosed tags
- [ ] the github projects need to be finalized (blocked on adobe/developer.adobe.com-planning#87)
- [x] add developer newsletter section

## Desktop View

Haven't even gotten there yet!

### CODE REVIEW

**Comment by stevengill** [comment]
So much simpler

**Comment by filmaj** [comment]
This could probably be given a review. Only mobile so far. Do we want to include desktop fidelity as part of this?

File: htdocs/topnav.css [hunk 1]
```diff
@@ -8,21 +8,33 @@
 .main-nav {
         list-style-type: none;
         padding-left: 20px;
+        list-style-type: none;
```
Comment (stevengill) [inline htdocs/topnav.css]: is this a dupe to line 9?

File: htdocs/topnav.css [hunk 2]
```diff
@@ -8,21 +8,33 @@
...
+        display: none;
 }
 
 .nav-links,
 .logo {
         text-decoration: none;
         color: rgba(2, 2, 2, 0.7);
 }
+.logo {
```
Comment (stevengill) [inline htdocs/topnav.css]: why not combine this with the logo selector in line 16?

File: htdocs/topnav.css (hunk 1 above)
Comment (filmaj) [reply htdocs/topnav.css]: lol yes it is! hahaha

File: htdocs/topnav.css (hunk 2 above)
Comment (filmaj) [reply htdocs/topnav.css]: wow well done fil. its like i cant see 3 lines above

**Comment by cassiemc** [comment]
yay, love to see this separated out. lmk if/when you want a design review.

**Comment by filmaj** [comment]
@cassiemc for sure! Maybe when we inch closer to landing the 'faithful representation' ?

**Pull request merged**